ElrondNetwork/elrond-go
# Fixed elastic search errors

- fixed elasticsearch errors that appeared when the templates files were missing but the indexer was disabled

Testing procedures:
- temporary remove elastic search templates file from config, start the node with the elastic search disabled, the node should start, no errors should appear (`grep -R ERROR` should return no result)
- temporary remove elastic search templates file from config, start the node with the elastic search enabled, the node should error stating that a .json file is missing from the elastic search templates directory. The node should stop after this error.
